pkg/controller/certificate-shim/helper.go

```diff
@@ -69,6 +69,10 @@ func translateAnnotations(crt *cmapi.Certificate, ingLikeAnnotations map[string]
 		crt.Spec.CommonName = commonName
 	}
 
+	if altNames, found := ingLikeAnnotations[cmapi.AltNamesAnnotationKey]; found {
+		crt.Spec.DNSNames = strings.Split(altNames, ",")
```

Comment:
I think we might need CSV encoding (just like in https://github.com/cert-manager/cert-manager/blob/master/internal/controller/certificates/secrets.go#L75)